src/NuGet.Core/NuGet.CommandLine.XPlat/Utility/ProjectPackagesPrintUtility.cs

```diff
@@ -119,6 +119,9 @@ internal static class ProjectPackagesPrintUtility
             var autoReferenceFound = false;
             var deprecatedFound = false;
 
+            // Exclude known project references from the listing.
+            packages = packages.Where(p => p.Type != "project");
```

Comment:
Use LibraryType.Project
```suggestion
            packages = packages.Where(p => p.Name != LibraryType.Project);
```

That's what the original model LockFileTargetLibrary uses.